Functions/xASL_bids_BIDSifyASLJSON.m

```diff
@@ -173,13 +173,24 @@
 end
     
 %% 7. Check for Hadamard sequence
+
+% Hadamard Check
+if isfield(jsonOut,'TimeEncodedMatrix') && ~isempty(jsonOut,'TimeEncodedMatrix') || ... % Should be 4, 8 or 12
+    isfield(jsonOut,'HadamardMatrixType') % Natural or walsh
```

Comment:
Should `HadamardMatrixType` not be `TimeEncodedType` or `TimeEncodedMatrixType`? Is this synced with OSIPI TF4.1?

Reply:
That's probably a question for @jan-petr or @amahroo.

Reply:
Conclusion dev meeting: `HadamardMatrixType `-> `TimeEncodedMatrixType`

Reply:
and HadamardMatrixSize -> TimeEncodedMatrixSize

Reply:
Solved 👍